docs/dev/cli/saucectl/docker/push.md

```diff
@@ -0,0 +1,93 @@
+---
+id: push
+title: saucectl docker push
+sidebar_label: saucectl docker push
+---
+
+Push a Docker image to the Sauce Labs Container Registry.
+
+Explore [additional details](/docs/orchestrate/saucelabs-private-registry.md) about the Sauce Labs Container Registry.
+
+## Usage
+
+```bash
+$ saucectl docker push image_name [flags]
+```
+
+:::note
+Ensure your `image_name` adheres to our Container Registry pattern.
```

Comment:
Is the naming pattern documented somewhere that we can link to here?

Reply:
I don't think there is any documented pattern. The only related documentation is contacting customer service 😂

Reply:
derp.

Should we just remove this warning then? What's the use in telling the reader that `image_name` has to follow a specific pattern if we're not going to tell them either what the pattern is or what to do if they encounter an error due to how they've named their image.

Reply:
Sounds fair!